source/extensions/filters/http/ext_proc/config.h

```diff
@@ -21,6 +21,7 @@ class ExternalProcessingFilterConfig
 
 private:
   static constexpr uint64_t DefaultMessageTimeoutMs = 200;
+  static constexpr uint64_t DefaultMaxMessageTimeoutMs = 10000;
```

Comment:
I wonder if DefaultMaxMessageTimeoutMs should default to zero. That way, this new functionality is disabled by default. If somebody wants to allow an ext_proc server to use this new functionality they would need to configure the ext_proc filter with a max_message_timeout > 0. Otherwise existing users of ext_proc will get this behavior enabled by default, which could be unexpected.

Reply:
Good idea.  Changed the default max to be zero to disable the new timeout API by default.